image/templates/helm/shared/templates/02-scanner-v4-07-matcher-deployment.yaml

```diff
@@ -107,9 +107,6 @@ spec:
         - name: config-volume
           mountPath: /etc/scanner
           readOnly: true
-        - name: proxy-config-volume
```

Comment:
There is no reason for the Matcher to talk to anything externally, so I removed this. However, is it possible someone makes proxies for inter-service communication? Perhaps? Maybe I should put this back to be safe.

Reply:
Going to put this back to be safe